src/panel/components/Pane.tsx

```diff
@@ -5,17 +5,30 @@ import React, {
   useMemo,
   MouseEventHandler,
   ComponentProps,
+  useRef,
 } from "react";
 import styled from "styled-components";
 import { useOrientationWatcher } from "../hooks";
 
-const PaneRoot: FC<ComponentProps<typeof PaneContainer>> = ({
+interface OverrideProps {
+  forcedOrientation?: { isPortrait: boolean };
+  initSize: { x: number; y: number };
+}
+
+const PaneRoot: FC<ComponentProps<typeof PaneContainer> & OverrideProps> = ({
```

Comment:
I'm sorry you had to deal with this component - we've even gone as far as to use some redundancy in other parts of the app because sharing it was causing us some problems (see [here](https://github.com/FormidableLabs/urql-devtools/blob/58f719b63600f08cf9c52b580a46b7accf36bad0/src/panel/pages/events/components/TimelinePane/TimelinePane.tsx#L132) and [here](https://github.com/FormidableLabs/urql-devtools/blob/58f719b63600f08cf9c52b580a46b7accf36bad0/src/panel/pages/request/components/Response.tsx#L124).